Wox.Plugin.Translate/GoogleTranslatePlugin.cs

```diff
@@ -293,45 +284,36 @@ public void Init(PluginInitContext context)
             _context = context;
         }
 
-        public static int topThreadId;
         public List<Result> Query(Query query)
         {
-            topThreadId = Thread.CurrentThread.ManagedThreadId;
             _context.API.StartLoadingBar();
             var list = new List<Result>();
-            var actionParameters = query.ActionParameters;
-            if (actionParameters.Count <= 2)
+
+
+            if (query.ThirdSearch == string.Empty || query.ThirdSearch == " ")
             {
                 for (var index = 0; index < Languages.GetLength(0); ++index)
                 {
                     var str = Languages[index, 0];
                     var countrycode = Languages[index, 1];
-                    list.Add(new Result
-                    {
-                        Title = str,
-                        SubTitle = countrycode,
-                        IcoPath = "Images\\pic.png",
-                        Action = e =>
+                        list.Add(new Result
                         {
-                            _context.API.ChangeQuery(query.RawQuery + countrycode + " ", false);
-                            return false;
-                        }
-                    });
+                            Title = str,
+                            SubTitle = countrycode,
+                            IcoPath = "Images\\pic.png",
+                            Action = e =>
+                            {
+                                _context.API.ChangeQuery(query.RawQuery + countrycode + " ");
+                                return false;
+                            }
+                        });
                 }
             }
-            else if (actionParameters.Count > 2)
-            {
-                var input = actionParameters[2];
-                var from = actionParameters[0];
-                var to = actionParameters[1];
-                input = String.Join(" ", actionParameters.Skip(2).ToArray());
-                Thread.Sleep(500);
```

Comment:
Will you update this PR or I shall merge it manually into a different branch and apply the changes? Thanks.